templates/base/server/.hathora/methods.ts.hbs

```diff
@@ -25,6 +26,7 @@ export interface Context {
   ) => void;
 }
 export interface Methods<T> {
+  stateChangeOptions?: Options & {pathAsArray?: false};
```

Comment:
what is the `& {pathAsArray?: false}` part for?

Reply:
That's how onChange accepts their options argument. It doesn't pass type check without it.

Reply:
https://github.com/sindresorhus/on-change/blob/main/index.d.ts#L254

Reply:
It looks like there is more to the story actually, there are overloads that I don't quite understand yet how to deal with.
https://github.com/sindresorhus/on-change/blob/main/index.d.ts#L267
https://github.com/sindresorhus/on-change/blob/main/index.d.ts#L280

Reply:
Ok, I figured it out. Since Hathora is the one using the `onChange` callback it doesn't affect anyone but we do need to choose whether or not we want the `onChange` callback to receive paths as an array or a string (even though we are not using the paths argument in the `onChange` callback). That choice will determine what type to use for `stateChangeOptions`.

Reply:
I chose `pathAsArray: true` for now since it does not matter and fixed the `stateChangeOptions` type so it's just `Options` now.